templates/default/container-executor.cfg.erb

```diff
@@ -1,2 +1,30 @@
+yarn.nodemanager.local-dirs=<%= node['hops']['tmp_dir'] %>/nm-local-dir
+yarn.nodemanager.log-dirs=<%= node['hops']['logs_dir'] %>/userlogs
 yarn.nodemanager.linux-container-executor.group=<%= @hops_group %>
 min.user.id=0
+banned.users=<%= node['hops']['hdfs']['user'] %>,<%= node['hops']['yarn']['user'] %>,<%= node['hops']['mr']['user'] %>,bin
+min.user.id=50
+[docker]
+module.enabled=true
+docker.binary=/usr/bin/docker
```

Comment:
passing the result of `which docker` would be safer here.

Reply:
will do.